actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -419,7 +423,7 @@ private void uploadCsvToDB(
     try {
       map.put(JsonKey.DATA, mapper.writeValueAsString(dataMapList));
     } catch (IOException e) {
-      ProjectLogger.log(e.getMessage(), e);
+      ProjectLogger.log("Exception while converting map to string " + e.getMessage(), e);
```

Comment:
From:
"Exception while converting map to string "

To:
"BulkUploadManagementActor:uploadCsvToDB: Exception while converting map to string: "